frb_rust/src/thread.rs

```diff
@@ -1,4 +1,32 @@
-pub const WORKERS_COUNT: usize = 4;
+#[cfg(not(wasm))]
+fn get_thread_count() -> usize {
+    #[allow(unused_mut, unused_assignments)]
+    let mut thread_count: usize = 4;
+    #[cfg(feature = "thread-count-single")]
+    {
+        worker_count = 1;
+    }
+    #[cfg(feature = "thread-count-max")]
```

Comment:
uber-nit: would be great to emit compile-time error if user wrongly specifies both features

Reply:
Okay :) That wouldn't be hard